l3kernel/l3pdf.dtx

```diff
@@ -97,15 +94,15 @@
 %   \end{itemize}
 % \end{function}
 %
-% \begin{function}[EXP, added = 2019-06-02]{\pdf_object_ref:n}
+% \begin{function}[EXP, added = 2021-02-10]{\pdf_object_ref:n}
 %   \begin{syntax}
 %     \cs{pdf_object_ref:n} \Arg{object}
 %   \end{syntax}
 %   Inserts the appropriate information to reference the \meta{object}
 %   in for example page resource allocation
 % \end{function}
 %
-% \begin{function}[added = 2019-06-02]
+% \begin{function}[added = 2021-02-10]
 %   {\pdf_object_now:nn, \pdf_object_now:nx}
```

Comment:
From today meeting: This function should be  named `\pdf_object_unnamed_write:nn`